src/yubikey-invalidation.coffee

```diff
@@ -33,7 +33,7 @@ module.exports = (robot) ->
       if res.statusCode != 200
         msg.reply "#{messagePrefix} I tried to invalidate that OTP for you, but I got a #{res.statusCode} error from the server 😢"
       else
-        msg.reply "#{messagePrefix} I went ahead and invalidated that OTP for you 🔒"
+        msg.reply "#{messagePrefix} I went ahead and invalidated that OTP for you 🔒 You can disable accidental OTP triggers via https://support.yubico.com/hc/en-us/articles/360013714379-Accidentally-Triggering-OTP-Codes-with-Your-Nano-YubiKey"
```

Comment:
Minor tweak; feel free to ignore (from [prior art](https://github.com/hubot-scripts/hubot-gh-issues/blob/719e93350eae312d111ccf545e4cd945d70e76f7/index.coffee#L20)):

```suggestion
        msg.reply "#{messagePrefix} I went ahead and invalidated that OTP for you 🔒\nYou can disable accidental OTP triggers via https://support.yubico.com/hc/en-us/articles/360013714379-Accidentally-Triggering-OTP-Codes-with-Your-Nano-YubiKey"

```